web/themes/custom/my_barrio/templates/comment.html.twig

```diff
@@ -0,0 +1,99 @@
+{#
```

Comment:
Update the template name to be more specific. This will override all of the comment section, try to use different template so that only the section you are using is updated and the other comment section is unchanged.

Reply:
Changed the file name to comment--field-comments-news.html.twig. It will change comments only on news content type.